osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs

```diff
@@ -107,7 +110,10 @@ public void Update(Playfield playfield)
             void handleHitCircle(DrawableHitCircle circle)
             {
                 if (!circle.HitArea.IsHovered)
+                {
+                    playfield.DisplayJudgements.Value = false;
```

Comment:
not sure using this to hide misses is correct/a good idea, either. in an ideal world i imagine `DrawableHitObject.DisplayResult` should be used for this somehow.